.github/workflows/arm-template-test.yml

```diff
@@ -53,4 +53,14 @@ jobs:
           quiet: false
           framework: arm
           output_format: cli
-          output_bc_ids: true
\ No newline at end of file
+          output_bc_ids: true
+
+  package-version-test:
```

Comment:
lets  create a separate github action for checking dependencies 
Also we need to do it for all 4 folders for both the solutions

Reply:
there are 3 folders, I have created a separate workflow file. Also added a matrix for jobs